common/config.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright 2019 The Kubeedge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/kubeedge/kubeedge/cloud/pkg/controller/constants"
+)
+
+//KubeEdgeCntrllerConfig has all the below fields; (data taken from "github.com/kubeedge/kubeedge/cloud/pkg/controller/config/kube.go")
+//Master is the url of edge master(kube api server)
+//KubeConfig is the config used connect to edge master
+//Namespace is the namespace to watch(default is NamespaceAll)
+//ContentType is the content type communicate with edge master(default is "application/vnd.kubernetes.protobuf")
+//QPS is the QPS communicate with edge master(default is 100.0)
+//Burst default is 10
+//NodeFrequency is the time duration for update node status(default is 20s)
+type KubeEdgeCntrllerConfig struct {
```

Comment:
nit: Cntrlller -> Ctrl ?

Reply:
Ctrl means more or Control, I want to indicate Controller and current way will be better to say it.

Reply:
Lets make it Controller then

Reply:
Done